common/display/vblankeventhandler.cpp

```diff
@@ -114,6 +123,8 @@ void VblankEventHandler::HandleRoutine() {
   int ret = drmWaitVBlank(fd, &vblank);
   if (!ret)
     HandlePageFlipEvent(vblank.reply.tval_sec, (int64_t)vblank.reply.tval_usec);
+
+  SetDrmMaster();
```

Comment:
dont think it is a good idea to check and set drm master in vblank handler coz every display has a vblank handler and the drm master is gpu device-wide related.